File: 크기가 작은 부분 문자열.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string t, string p) {
    int answer = 0;
    int l=p.length();
    for(int i=0;i<t.length()-(l-1);i++)
    {
        string temp;
        for(int j=i;j<i+l;j++)
        {
            temp.push_back(t[j]);
        }
        
        if(temp<=p)
        {
            answer++;
        }
    }
    return answer;
}

File: Java/LV1/크기가 작은 부분 문자열.java
import java.util.*;

class Solution {
    public int solution(String t, String p) {
        int answer = 0;
        
        for(int i=0;i<=t.length()-p.length();i++){
            String k = t.substring(i,i+p.length());
            
            //System.out.println(Integer.parseInt(k));
            if(Long.parseLong(k)<=Long.parseLong(p)){
                answer++;
            }
        }
       
        return answer;
    }
}
